### DISCUSSION TITLE
Update Timezone

### DISCUSSION THREAD

**Discussion by TheBrambleShark** [firefly-iii/firefly-iii#8673] [Firefly III]
Is there a way to update the current timezone without losing any data?

**Comment by JC5** [comment]
I'm not sure what you are afraid to lose?

**Reply by TheBrambleShark** [reply]
Transaction data, accounts, etc.
Though I suppose that's stored in another container.

Do I just drop the main app container, change the docker-compose.yml file, and use `docker up` again?

**Reply by JC5** [reply]
It should be enough to stop/start, yes :)

**Comment by JC5** [comment]
One thing is important: Firefly III does not store the time zone with the transactions; so if you change the timezone your current set of transactions may jump to another moment / day.
